inventory/vagrant/group_vars/webserver/apache.yml

```diff
@@ -1,5 +1,6 @@
 ---
 
+apache_listen_ip: localhost
```

Comment:
Adding this somehow messes with the virtual host file and now root is /var/www/html instead of /var/www/html/drupal/web

Reply:
Likely a geerlingguy artefact. Any reason why we want it there in the first place? Not having it in the past hasn't caused any trouble that I'm aware of.

Reply:
I think this will come up fine if we change this to: 
```
apache_listen_ip: "*"
```

which is the default, so we could probably not set it as well.

Reply:
I stand corrected, we do [need the variable set](https://github.com/Islandora-Devops/claw-playbook/pull/66/files#diff-c1141c686f17f76985d20bc3dc07db39R8) for this PR. I'm testing @jonathangreen 's suggestion now.